tests/config/test_configtypes.py

```diff
@@ -1124,15 +1151,16 @@ class FontTests(unittest.TestCase):
     }
     INVALID = ['green "Foobar Neue"', 'italic green "Foobar Neue"',
                'bold bold "Foobar Neue"', 'bold italic "Foobar Neue"'
-               'bold', '10pt 20px "Foobar Neue"']
+                                          'bold', '10pt 20px "Foobar Neue"']
```

Comment:
Oh, weird! I don't think that `bold` is supposed to be there, as the aim of the test is to test the `bold italic ...`. I'll push a commit which fixes it and uses one line per test because the quotes quickly get confusing.